Dockerfile

```diff
@@ -1,31 +1,35 @@
 # A build environment for the Hosted Graphite agent.
 # Because we use PyInstaller, we need to build against the oldest glibc we
 # intend to support: that's 2.12, available in CentOS 6.
```

Comment:
This comment must be updated or removed, don't you agree?

Reply:
Updated
![image](https://user-images.githubusercontent.com/40685761/155692078-f65c8f18-dc12-4e1c-bf6c-b8af4ab13835.png)